tests/unit/Term/AliasGroupListTest.php

```diff
@@ -191,12 +184,10 @@ public function testGivenEmptyGroup_setGroupRemovesGroup() {
 			new AliasGroup( 'en', array( 'foo' ) ),
 		) );
 
-		$expectedList = new AliasGroupList();
-
 		$list->setGroup( new AliasGroup( 'en' ) );
 		$list->setGroup( new AliasGroup( 'de' ) );
 
-		$this->assertEquals( $expectedList, $list );
+		$this->assertTrue( $list->isEmpty() );
```

Comment:
You know how `assertEquals` works? It compares **all** implementation details of the two objects. How is this better? All this test should care about is if the group is removed. It should not depend on everything else that can possibly happen in the object.